spec/services/idv/phone_step_spec.rb

```diff
@@ -37,7 +37,7 @@
 
       subject.submit(phone: good_phone)
 
-      expect(subject.async_state.status).to eq :done
+      expect(subject.async_state.done?).to eq true
```

Comment:
as a fun Rspec-ism in case we wanted, you can do:

```suggestion
      expect(subject.async_state).to be_done
```

Reply:
thank you, TIL!